src/test/java/eu/deltasource/model/BankInstitutionTest.java

```diff
@@ -0,0 +1,25 @@
+package eu.deltasource.model;
+
+import eu.deltasource.exception.InvalidInputException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class BankInstitutionTest {
+
+    @Test
+    void testIfABlankBankNameIsEntered() {
+        //GIVEN
+        Assertions.assertThrows(InvalidInputException.class, () -> {
+            BankInstitution bankInstitution = new BankInstitution("    ", "ivan 2");
```

Comment:
Ok, lets say I see this for the first time, without knowing the other code.
How can you make it so I understand why are using empty string with differerent length ?